app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -266,9 +255,8 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         }
         val loginsNameKey = getPreferenceKey(R.string.pref_key_sync_logins)
         findPreference<CheckBoxPreference>(loginsNameKey)?.apply {
-            isVisible = FeatureFlags.logins
             isEnabled = syncEnginesStatus.containsKey(SyncEngine.Passwords)
-            isChecked = syncEnginesStatus.getOrElse(SyncEngine.Passwords) { false }
+            isChecked = syncEnginesStatus.getOrElse(SyncEngine.Passwords) { true }
```

Comment:
Sorry, I should have updated the bug! Since we have all the biometrics/pin checking set up now and we are removing the nightly flag, we are changing the default for syncing logins to true. cc @liuche to confirm

Reply:
Nvm changing this is dependent on https://github.com/mozilla-mobile/fenix/issues/6704 being on for release users